packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -306,6 +312,12 @@ exports.onCreateNode = (
     navLevels: node.frontmatter.navLevels
       ? Number(node.frontmatter.navLevels)
       : 3,
+    showTimeToRead: node.frontmatter.showTimeToRead
+      ? Boolean(node.frontmatter.showTimeToRead)
+      : false,
+    setTimeToRead: node.frontmatter.setTimeToRead
+      ? Number(node.frontmatter.setTimeToRead)
```

Comment:
Did you test what happens if you put something that does not parse into a Number? It would be good if the dev server would not crash in the situation but rather just ignore it